sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileMetadataStructSuite.scala

```diff
@@ -524,10 +525,11 @@ class FileMetadataStructSuite extends QueryTest with SharedSparkSession {
         .select("*", "_metadata")
         .writeStream.format("json")
         .option("checkpointLocation", dir.getCanonicalPath + "/target/checkpoint")
+        .trigger(Trigger.Once())
```

Comment:
nit: Please change to `Trigger.AvailableNow()` and see whether this breaks the test or not. We are going to produce deprecated warning Trigger.Once() from Spark 3.4.0.